docs/src/parameter_distributions.md

```diff
@@ -21,119 +26,106 @@ dict_1 = Dict("distribution" => distribution_1, "constraint" => constraint_1, "n
 dict_2 = Dict("distribution" => distribution_2, "constraint" => constraint_2, "name" => name_2)
 prior = ParameterDistribution( [dict_1, dict_2] )
 ```
+
 We provide many examples in the package `examples/`, the unit tests found in `test/ParameterDistributions/runtests.jl`, and in the documentation below.
 
-# A simple example:
-Task: We wish to create a prior for a one-dimensional parameter. Our problem dictates that this parameter is bounded between 0 and 1. Prior knowledge dictates it is around 0.7. The parameter is called `point_seven`.
+### Constrained\_gaussian constructor
 
-Solution: We should use a Normal distribution with the predefined "bounded" constraint.
+The constructors described above can be used to specify arbitrary prior distributions and constraints. However, the anticipated use case for EnsembleKalmanProcesses is one where inference is not strongly prior-dominated: i.e., the user only needs (or is only able to) specify the physical and likely ranges of prior parameter values at a rough, qualitative level.
+
+For this use case we provide the `constrained_gaussian` constructor as a streamlined syntax for specifying a special case of the prior. Because it finds the appropriate distribution and constraint automatically, we **highly** recommend users use this as a starting point, only using the general constructor when necessary.
 
-Let's initialize the constraint first,
-```julia
-constraint = bounded(0, 1)
-```
-This **automatically** sets up the following transformation to the  constrained space (and also its inverse)
-```julia
-transform_unconstrained_to_constrained(x) = exp(x) / (exp(x) + 1)
-```
-The prior should be around 0.7 (in the constrained space), and one can find that the push-forward of a particular normal distribution, namely, `transform_unconstrained_to_constrained(Normal(mean = 1, sd = 0.5))` gives a prior pdf with 95% of its mass between [0.5, 0.88].
-```julia
-distribution = Parameterized(Normal(1, 0.5))
-```
-Finally we attach the name
-```julia
-name = "point_seven"
-```
-and the distribution is created by either:
-```julia
-prior = ParameterDistribution(distribution, constraint, name)
-```
-or
 ```julia
-prior_dict = Dict("distribution" => distribution, "constraint" => constraint, "name" => name)
-prior = ParameterDistribution(prior_dict)
+prior_1 = constrained_gaussian("param_name", μ_1, σ_1, 0.0, Inf)
+prior_2 = constrained_gaussian("param_name", μ_2, σ_2, lower_bound, upper_bound)
+prior = combine_distributions( [prior_1, prior_2])
 ```
 
-```@setup zero_point_seven
-using Distributions
-using Plots
-Plots.default(lw=2)
+This constructs a prior distribution on the interval [`lower_bound`, `upper_bound`] with mean μ and standard deviation σ. The distribution itself is a Guassian "squashed" according to the interval chosen (e.g. `prior_1` is a log-normal.) See [Simple example - 1](@ref) for an example.
```

Comment:
Typo "Guassian"

Reply:
Good catch! Fixed in [9b09cea](https://github.com/CliMA/EnsembleKalmanProcesses.jl/pull/161/commits/9b09cea7032cefb92cd0332b9656b924152356c1)